doc/python/interconnectivity.rst

```diff
@@ -7,24 +7,28 @@ Interconnectivity
 
 .. class:: connection
 
-    Describes a connection between two cells, defined by source and destination end points (that is pre-synaptic and post-synaptic respectively),
-    a connection weight and a delay time.
+    Describes a connection between two cells, defined by source and destination end points (that is pre-synaptic and
+    post-synaptic respectively), a connection weight and a delay time.
 
-    The :attr:`dest` does not include the gid of a cell, this is because a :class:`arbor.connection` is bound to the destination cell which means that the gid
-    is implicitly known.
+    The :attr:`dest` does not include the gid of a cell, this is because a :class:`arbor.connection` is bound to the
+    destination cell which means that the gid is implicitly known.
 
     .. function:: connection(source, destination, weight, delay)
 
         Construct a connection between the :attr:`source` and the :attr:`dest` with a :attr:`weight` and :attr:`delay`.
 
     .. attribute:: source
 
-        The source end point of the connection (type: :class:`arbor.cell_member`, which can be initialized with a (gid, index) tuple).
+        The source end point of the connection (type: :class:`arbor.cell_global_label`, which can be initialized with a
+        (gid, label) or a (gid, label, policy) tuple. If the policy is not indicated, the default
```

Comment:
The constructor doesn't have a 3-tuple form as currently defined, so it'd be `(gid, (label, policy))`.
Given that gap junctions currently require a univalent policy, should the default still be round_robin? Another option might be to force an explicit policy, even though it's pretty verbose.

Reply:
I think we can change the default to assert_univalent. Do yo

Reply:
I think it's fine to default to univalent. The verbosity of specifying a policy explicitly (and similarly with other enum-using interfaces) is something we can look at again in the future.